cmd/cli/uninstall_mesh_test.go

```diff
@@ -196,11 +201,9 @@ func TestUninstallCmd(t *testing.T) {
 				mem.SetNamespace(test.meshNamespace)
 			}
 
-			if test.meshExists {
-				rel := release.Mock(&release.MockReleaseOptions{Name: test.meshName})
-				err := store.Create(rel)
-				Expect(err).To(BeNil())
-			}
+			rel := release.Mock(&release.MockReleaseOptions{Name: testMeshName})
+			err := store.Create(rel)
+			Expect(err).To(BeNil())
```

Comment:
I know this is how it was before, but let's stick to using `testify` in non-ginkgo tests.